### DISCUSSION TITLE
Linux install instructions

### DISCUSSION THREAD

**Discussion by elch01** [seerr-team/seerr#410] [General]
Hi,

I can see that the instructions for the Linux installation are not very clear, e.g see issue #391 or #386 (the /opt folder should maybe be put as a default?)
I have myself had many issues setting up the application on Ubuntu 22.04 such as yarn from the apt package installer not being compatible and having to use npm and symlinking yarn from /usr/local/lib/node_modules/yarn/bin/yarn to /bin/yarn.
More issues such as "The engine "node" is incompatible with this module.", could we maybe get some more verbose instructions? distro specific instructions?

I'm sure the installation process is different on other distros (such as the AUR on Arch), but these issues seem to affect both Debian and Ubuntu from my experience and i have yet found a way around the issues with incompatibility.

Has anyone else encountered these issues?

**Comment by fallenbagel** [comment]
Maybe the instructions need more detail yeah but the instructions would work on any Linux distros (even in arch). I'll try making a more detailed installation instruction i guess for the manual method (though docker is more recommended(

The yarn in apt is cmdlet not the yarn package manager.

You need atleast node LTS version as stated in the instructions, the apt manager only provides I think v14. You can get node LTS version instructions on their website or using nvm. Then install yarn using `npm install -g yarn`.

> the /opt folder should maybe be put as a default?

I'm not sure what you mean by this?

**Reply by elch01** [reply]
Awesome.

> The yarn in apt is cmdlet not the yarn package manager.
> You need atleast node LTS version as stated in the instructions

A notice about this needs to be added, its really confusing when the instructions just say "Nodejs (atleast LTS version)" & "yarn".
You'd usually expect the packages to come from the package manager, not npm as there is no mention of this.

>  I'm not sure what you mean by this?

What i mean is that a recommendation should be to git clone the files to /opt in the instructions, as this seems to be the usual install folder for the *arr applications (unless installed from the package manager).
This should avoid users just putting all the application files in their home folder (a common occurrence), /home should only contain the users own files and not applications.

EDIT: I've added a pull request, see if that looks good (#411) :+1:
